### PR TITLE
use nodesource repo to get newer node versions

### PR DESCRIPTION
**Pull request by evantahler**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,14 @@
 FROM ubuntu:trusty
-MAINTAINER arthur@caranta.com
+MAINTAINER evan@evantahler.com
 
 RUN apt-get update -y
-RUN apt-get install -y npm && apt-get clean && rm -rf /var/lib/apt/lists/*
-RUN npm install elasticdump -g
+RUN curl -sL https://deb.nodesource.com/setup_4.x | sudo -E bash -
```
Comment (canadatom) [inline Dockerfile]: curl needs to be installed first.

Comment (evantahler) [inline Dockerfile]: thanks! try now.

**Comment by canadatom** [comment]
This is working one for me, put all RUN together to optimize the build performance, and remove the ln since it's already exist in the path.
```
FROM ubuntu:trusty
MAINTAINER evan@evantahler.com

RUN apt-get update -y && \
    apt-get install -y curl && \
    curl -sL https://deb.nodesource.com/setup_4.x | sudo -E bash - && \
    apt-get install -y nodejs && \
    apt-get clean && \
    rm -rf /var/lib/apt/lists/*

RUN npm install elasticdump -g

#And finally use elasticdump as the entrypoint
ENTRYPOINT ["/usr/bin/elasticdump"]
```

**Comment by evantahler** [comment]
thanks!
up on NPM

**Pull request merged**